src/main/java/edu/harvard/iq/dataverse/search/SolrSearchResult.java

```diff
@@ -417,7 +417,7 @@ public JsonObjectBuilder getJsonForMyData() {
                 .add("is_unpublished_state", this.isUnpublishedState())
                 .add("is_published", this.isPublishedState())
                 .add("is_deaccesioned", this.isDeaccessionedState())
-                .add("date_to_display_on_card", this.dateToDisplayOnCard);
+                .add("date_to_display_on_card", DateUtil.formatDate(this.releaseOrCreateDate));
```

Comment:
@JayanthyChengan This change is still missing (and should be last one!): change this line to call getDateToDisplayOnCard(), that way the logic is in only one place (that method).

Reply:
that's right.  Thanks.